storm-core/src/jvm/org/apache/storm/daemon/StormCommon.java

```diff
@@ -77,7 +78,7 @@ public static StormCommon setInstance(StormCommon common) {
         return oldInstance;
     }
 
-    private static final Logger LOG = LoggerFactory.getLogger(StormCommon.class);
+    private static final Logger LOG = getLogger(StormCommon.class);
```

Comment:
lets keep it to LoggerFactory.getLogger given that we are using the same way everywhere else.